python_backend/api/v1/endpoints/conversion.py

```
@@ -0,0 +1,82 @@
+3. 공통 응답 메세지는 모델로 정의 
+4. 의미 있는 status code 분리 
+5. pydantic 모델이 request body 에서 명시적임을 보장 
+"""
+
+from typing import Annotated
+from fastapi import APIRouter, Depends, HTTPException
+from dependency_injector.wiring import inject, Provide
+
+from core.container import Container
+from services.conversion_service import ConversionService
+from services.conversion_service import ConversionService
```

Ruff (0.12.2)

Redefinition of unused `ConversionService` from line 19

Remove definition: `ConversionService`

(F811)